WrightTools/units.py

```diff
@@ -152,43 +129,43 @@ def get_symbol(units) -> str:
     string
         LaTeX formatted symbol.
     """
-    if kind(units) == "energy":
-        d = {}
-        d["nm"] = r"\lambda"
-        d["wn"] = r"\bar\nu"
-        d["eV"] = r"\hslash\omega"
-        d["Hz"] = r"f"
-        d["THz"] = r"f"
-        d["GHz"] = r"f"
-        return d.get(units, "E")
-    elif kind(units) == "delay":
+    dimensionality = str(ureg.Unit(units).dimensionality)
+    if dimensionality == "[length]":
+        return r"\lambda"
+    if dimensionality == "1 / [length]":
+        return r"\bar\nu"
+    if dimensionality == "[length] ** 2 * [mass] / [time] ** 2":
+        return r"\hslash\omega"
+    if dimensionality == "1 / [time]":
+        return "f"
+    if dimensionality == "[time]":
         return r"\tau"
-    elif kind(units) == "fluence":
+    if dimensionality == "[mass] / [time] ** 2":
         return r"\mathcal{F}"
-    elif kind(units) == "pulse_width":
-        return r"\sigma"
-    elif kind(units) == "temperature":
-        return r"T"
+    if dimensionality == "[temperature]":
+        return "T"
     else:
         return kind(units)
```

Comment:
```suggestion
        return "{:L}".format(ureg.Unit(units))
```

Reply:
https://pint.readthedocs.io/en/0.7.2/tutorial.html#string-formatting

Reply:
I do not think this is what we want...

That will result in (in Dariens example of Tesla) `tesla (T)`

I think perhaps the right call is to return None, and then place the `axis.name` in the string in the axis class...

Additionally, pint's Latex formatting uses `mathrm`, where we have historically prefered `mathsf`